libs/soo_color_palettes.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include <vector>
+
+namespace soo {
+namespace colors {
+
+using Palette = std::vector<int>;
+
+const Palette procreate_flourish{
```

Comment:
maybe add a test function that displays a square for each color to add the color palette to the documentation (libs README)? (nice to have)